Sources/ProjectDescription/Config.swift

```diff
@@ -14,6 +14,28 @@ public struct Config: Codable, Equatable {
             case targets([TargetReference])
         }
 
+        /// Contains options for autogenerated targets
+        public enum AutogenerationOptions: Codable, Equatable {
+            /// Contains options for test options within autogenerated targets
+            public struct TestingOptions: OptionSet, Codable, Equatable {
+                public let rawValue: Int
+
+                public init(rawValue: Int) {
+                    self.rawValue = rawValue
+                }
```

Comment:
Can't these be private? if yes, it should be to avoid allowing to instantiate invalid options

Reply:
No, these are required to be public because of the `OptionSet` protocol. I tried to add protocol conformance in an extension and to mark the init as unavailable but no luck so left these public

Reply:
Yeah, the methods conforming to a protocol will always have to have the same visibility of the conforming type itself.